src/main/java/fr/adrienbrault/idea/symfony2plugin/dic/inspection/CaseSensitivityServiceInspection.java

```diff
@@ -91,6 +92,11 @@ public void visitElement(PsiElement psiElement) {
 
     private void phpVisitor(final @NotNull ProblemsHolder holder, @NotNull PsiFile psiFile) {
 
+        /* skip inspections since it doesn't make sense in SF 4, where service IDs are often FQNs */
+        if (SymfonyUtil.isVersionGreaterThen(psiFile.getProject(), "3.4.999")) {
```

Comment:
in yml / xml there is a check if is are class names, maybe we can just is this here also